client/my-sites/plans/jetpack-plans/jetpack-complete-page/index.tsx

```diff
@@ -67,6 +68,7 @@ const JetpackCompletePage: React.FC< Props > = ( {
 							all CRM extensions, and extra storage for backups and video.
 						</p>
 					</>
```

Comment:
Not super related to your PR itself, but we have an unnecessary React Fragment here. We can simplify the markup by removing it, as follows:

```tsx
<JetpackRnaDialogCard ...stuff>
  <h1 ...stuff>
    ...
  </h1>
  <p ...stuff>
    ...
  </p>
  <CtaButtons />
</JetpackRnaDialogCard>
```